exchangelib/folders/known_folders.py

```diff
@@ -50,7 +50,8 @@ class DeletedItems(Folder):
 
 class Messages(Folder):
     CONTAINER_CLASS = 'IPF.Note'
-    supported_item_models = (Message, MeetingRequest, MeetingResponse, MeetingCancellation)
+    supported_item_models = (Message, MeetingRequest, MeetingResponse, MeetingCancellation, CreateChange,
+                             UpdateChange, DeleteChange, ReadFlagChange)
```

Comment:
I don't think you need the `CreateChange, UpdateChange, DeleteChange, ReadFlagChange` added here

Reply:
test_properties still shows duplicate fields on `ReadFlagChange` though, any idea why?

Reply:
That's because these classes need a `__slots__` definition. For the classes that don't define additional fields, add: `__slots__ = tuple()`